website_product_subscription/controllers/subscribe.py

```diff
@@ -245,7 +245,7 @@ def _process_invoice_address(self):
 
         representative = params["sponsor_id"]
         inv_add_values = {
-            "parent_id": representative.id,
+            "parent_id": representative,
```

Comment:
are you sure ? normally a many2one field should take the id and not the whole record.

Reply:
It can take both :-)